storage/ast/node_set.go

```diff
@@ -52,7 +52,6 @@ func (node *AllOfSetExprNode) EvalBool(s Symbols) (result bool, err error) {
 	if err != nil || cursor == nil {
 		return
 	}
-	defer cursor.Close()
```

Comment:
i see a bunch of cursor.Close removed - which i expect is intentional but I want to make sure it's closed elsewhere. I bet it is - but i didn't see it in this function any more. Can you just comment on this to make sure it's 'a-ok'? :)

Reply:
Actually, I am thinking maybe it's because it's moved to bbolt - it's not needed?

Reply:
Yes, it wasn't needed. I had added an API for it, which just nulled some resources, but there's nothing that won't get GCed. Coming from a DB mindset, it felt like something that needed to be closed, but in bbolt that's not the case and I was finally able to transcend my database biases :)